Libs/MRML/Widgets/qMRMLChartWidget.cxx

```diff
@@ -161,3 +172,14 @@ vtkMRMLColorLogic* qMRMLChartWidget::colorLogic()const
   Q_D(const qMRMLChartWidget);
   return d->ChartView->colorLogic();
 }
+
+#ifndef MRML_WIDGETS_HAVE_WEBKIT_SUPPORT
+//---------------------------------------------------------------------------
+void qMRMLChartWidget::onAppAboutToQuit()
+{
+  Q_D(qMRMLChartWidget);
```

Comment:
May be sensible to also add:

```
d->ChartView->setVisible(false);
d->ChartView->setParent(0);
```

Reply:
Does it matter ? The whole application is shutting down, so that shouldn't change anything ?